IdentityCore/src/cache/mac/MSIDMacCredentialStorageItem.m

```diff
@@ -158,57 +176,41 @@ - (NSDictionary *)jsonDictionary
     return dictionary;
 }
 
-- (MSIDDefaultCredentialCacheKey *)getKeyForCredential:(MSIDCredentialCacheItem *)credential
+- (MSIDCacheKey *)createKeyFromCredential:(MSIDCredentialCacheItem *)credential
 {
-    MSIDDefaultCredentialCacheKey *credentialKey = [[MSIDDefaultCredentialCacheKey alloc] initWithHomeAccountId:credential.homeAccountId environment:credential.environment clientId:credential.clientId credentialType:credential.credentialType];
+    MSIDDefaultCredentialCacheKey *credentialCacheKey = [[MSIDDefaultCredentialCacheKey alloc] initWithHomeAccountId:credential.homeAccountId environment:credential.environment clientId:credential.clientId credentialType:credential.credentialType];
```

Comment:
nit: this code is duplicate with code in account credential cache. Can we combine those two?

Reply:
As discussed, created a helper method in MSIDCredentialCacheItem class to create a credential cache key from credential.